racetime/models/race.py

```diff
@@ -1276,6 +1284,7 @@ def partition(self):
                 allow_midrace_chat=False,
                 allow_non_entrant_chat=False,
                 chat_message_delay=self.chat_message_delay,
+                opened_by_bot=self.opened_by_bot,
```

Comment:
This shouldn't be set here IMO, since the partitioned race rooms aren't being created by the bot (at least, not necessarily).

Reply:
Done